src/frontend/src/common/components/library/Table/index.tsx

```diff
@@ -1,7 +1,35 @@
-import React from "react";
+import React, { ReactNode } from "react";
+import AutoSizer from "react-virtualized-auto-sizer";
+import { FixedSizeList } from "react-window";
+import { ITEM_HEIGHT_PX } from "../data_table";
+import { Row } from "./components/Row";
+import { Container, DataRows } from "./style";
 
-const Table = (): JSX.Element => {
-  return <div>beep</div>;
-};
+interface Props {
+  headers: ReactNode[];
+  rows: ReactNode[][];
+}
+
+const Table = ({ headers, rows }: Props): JSX.Element => (
+  <Container>
+    <Row cells={headers} isHeader />
+    <DataRows>
+      <AutoSizer>
+        {({ height, width }) => {
```

Comment:
i'm a little confused by this notation, where is the height and width coming from?

Reply:
This is a function definition, so whatever calls it needs to provide a height and width to this function.

In this case, AutoSizer requires a function the accepts a height/width as a child.

Reply:
i didn't see height and width in the props though for Table?

Reply:
They are provided by AutoSizer, which is not our component

Reply:
i get it now